Assets/LeapMotionModules/UI/LeapMotionHandsExtentionAPI/FingerType.cs

```diff
@@ -0,0 +1,38 @@
+﻿
+namespace Leap.Unity {
+
+  public static class FingerType {
```

Comment:
This whole class seems to be unused?  Furthermore, it seems to only mirror the Leap.Finger.FingerType enum, what is the reason it exists?